### PR TITLE
Add compat data for input-datetime-local

### PR DESCRIPTION
**Pull request by maboa**
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/datetime-local

### CODE REVIEW

File: html/elements/input/datetime-local.json [hunk 1]
```diff
@@ -0,0 +1,58 @@
+{
+  "html": {
+    "elements": {
+      "input": {
+        "input-datetime-local": {
+          "__compat": {
```
Comment (wbamberg) [inline html/elements/input/datetime-local.json]: As [before](https://github.com/mdn/browser-compat-data/pull/2198#issuecomment-395250429), I think this needs:

    "description": "<code>type=\"datetime-local\"</code>"

File: html/elements/input/datetime-local.json [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+              },
+              "edge": {
+                "version_added": "12"
+              },
+              "edge_mobile": {
+                "version_added": null
+              },
+              "firefox": {
+                "version_added": false,
+                "notes": "This feature is not implemented yet. See <a href='https://bugzil.la/888320'>bug 888320</a> and <a href='https://wiki.mozilla.org/TPE_DOM/Date_time_input_types'>TPE DOM/Date time input types</a>."
```
Comment (wbamberg) [inline html/elements/input/datetime-local.json]: I would omit "This feature is not implemented yet." as it is redundant. Also the project page link doesn't seem very active, it's probably better to just link to the bug:

    "See <a href='https://bugzil.la/888320'>bug 888320</a>."

File: html/elements/input/datetime-local.json [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+              },
+              "opera": {
+                "version_added": "11"
+              },
+              "opera_android": {
+                "version_added": "11"
+              },
+              "safari": {
+                "version_added": false,
+                "notes": "It is recognized but there is no UI."
```
Comment (wbamberg) [inline html/elements/input/datetime-local.json]: Oh, and I agree with [Daniel's comment on this](https://github.com/mdn/browser-compat-data/pull/2199#discussion_r194079457).

**Comment by maboa** [comment]
Made the changes as requested. Thanks!

**Pull request merged**